examples/sample_encryption.py

```diff
@@ -4,6 +4,9 @@
 from uid2_client import Uid2Client, IdentityScope
 from uid2_client import encrypt_data, decrypt_data
 
+# THIS FILE IS DEPRECATED!
+# if you would like to learn how to encrypt and decrypt UID2 advertising token, please
```

Comment:
# To learn how to encrypt and decrypt a UID2 advertising token, see sample_client.py.

Reply:
fixed